spec/features/post_and_view_messages_spec.rb

```diff
@@ -0,0 +1,39 @@
+require './lib/message'
+require './lib/user'
+require 'capybara/rspec'
+require './spec/web_helper'
+
+
+feature '1. Post Peep to Chitter' do
+  scenario 'Post new message and display it beneath' do
+    signup_steps
+    signin_steps
+    fill_in('content', :with => 'Peep peep')
+    click_button('Send')
+    expect(page).to have_content('Peep peep')
+  end
+end
+
+feature '2. View all Peeps' do
+  scenario 'Shows all peeps in reverse chronological order' do
+    signin_steps
+    fill_in('content', :with => 'Peep1')
+    click_button('Send')
+    fill_in('content', :with => 'Peep2')
+    click_button('Send')
+    fill_in('content', :with => 'Peep3')
+    click_button('Send')
+    expect(page).to have_content('Peep1')
+    expect(page).to have_content('Peep2')
+    expect(page).to have_content('Peep3')
+  end
+end
+
+feature '3. Timestamp' do
+  scenario 'See the time at which peep was made' do
+    signin_steps
+    fill_in('content', :with => 'Peep peep')
+    click_button('Send')
+    expect(page).to have_content('Peep peep')
+  end
+end
```

Comment:
Again I'll send you the timecop info so you can test for time something was created, as you said this test is pretty much the same as 3.
On a side note, like that you've numbered your tests, makes it really easy to refer to them so I'll be picking up this practice for myself in the future.